### ISSUE TITLE
Hiding Map features

### ISSUE THREAD

**Issue by mmd-osm** [openstreetmap/iD#7269]
Step 1: I have disabled, say _buildilngs_ in the map features panel some time ago and forgot about it.

![a1](https://user-images.githubusercontent.com/5842757/72670935-db9de780-3a43-11ea-8216-3d38ecb5126a.png)

Step 2: A bit later, I started adding a few new buildings which appear to be missing on the map. For some reason the _Map Features_ filtering doesn't kick and every building I add is happily shown on the map as if the filter wasn't active. This by the way makes it a bit more difficult to remember what I did earlier on in step 1.

![a3](https://user-images.githubusercontent.com/5842757/72670937-de98d800-3a43-11ea-8448-4910c249973d.png)

I see that the validator complains about those new buildings and flags them as duplicates (which is correct, as I added new buildings over existing ones, which have been hidden by the filter). However, when clicking on the warning, only the newly added house is highlighted, making it a bit difficult to figure out what the real issue is here.

![a2](https://user-images.githubusercontent.com/5842757/72670936-dccf1480-3a43-11ea-8625-1db4558c7cbf.png)

I'm not sure at which point the UI could be improved here. As it is right now, hiding Map features has slightly surprising side effects and somehow feels a bit inconsistent.

**Comment by maro-21** [comment]
You drew an area and probably added building=yes manually? Because when Building layers is hidden and you search the building preset, iD says that this layer is hidden.

**Comment by mmd-osm** [comment]
No, I searched for the building in the presets and added a preset for a building. This is in 3.x by the way.

**Comment by quincylvania** [comment]
Hmm we show this message in v3, but apparently not when changing an existing preset (e.g. Area). I'll fix that.

<img width="237" alt="Screen Shot 2020-01-20 at 9 34 12 AM" src="https://user-images.githubusercontent.com/2046746/72734617-18fb9580-3b68-11ea-8883-d23b30beddfe.png">
<img width="358" alt="Screen Shot 2020-01-20 at 9 34 06 AM" src="https://user-images.githubusercontent.com/2046746/72734618-18fb9580-3b68-11ea-9e97-03e15da4d544.png">

Let us know if you're not seeing this message in the toolbar with a layer disabled.

**Comment by mmd-osm** [comment]
I just double checked what exactly I did:

(1) HIde buildings in Map features
(2) Add Building as Area geometry (without initially defining what that area is)
(3) In the sidebar, I used the search function to locate "Building" and selected it.
(4) The area was turned into a building, I didn't see any popups that buildings were hidden at this point.

**Comment by k-yle** [comment]
I'm going to close this issue, since it's a bug in the v3 prototype version, which is (sadly) abandoned and [no longer deployed](https://github.com/openstreetmap/iD/commit/36f4eeb7e24ba3ffd16f0d48780f3c56d39f2bd1).
